tests/models/test_cleartasks.py

```diff
@@ -597,7 +597,7 @@ def test_dag_clear(self, dag_maker):
         ti0, ti1 = sorted(dr.task_instances, key=lambda ti: ti.task_id)
         ti0.refresh_from_task(task0)
         ti1.refresh_from_task(task1)
-        session.get(TaskInstance, ti0.key.primary).try_number += 1
+        session.get(TaskInstance, ti0.id).try_number += 1
```

Comment:
The reason for this is explained in [**Issues identified Point (2)**](https://github.com/apache/airflow/pull/43243#issuecomment-2439774584)